### ISSUE TITLE
Hue brightness 1 turns lights off

### ISSUE THREAD

**Issue by Glennmen** [zigpy/zha#217]
Applying brightness value 1 to a Hue light turns it off. In the UI it seems to be still "on" but the actual light is not on anymore.
![CleanShot 2024-09-30 at 20 04 59@2x](https://github.com/user-attachments/assets/9ffa6f02-88a7-4cd7-810b-f7d11bb4e910)

This is different behaviour than when I do the same to a Hue light that is still connected to my Hue bridge, there it seems to send a different brightness value to the actual lamp and it still turns on with a very low brightness.

The attribute value reported back in both cases is `brightness: 1` so I am assuming the bridge does some custom behaviour in case of this value.

**Comment by puddly** [comment]
Are you sure the light is not just very dim? Hue lights can get unusably dim.

**Comment by Glennmen** [comment]
@puddly I am pretty sure that isn't the case.
One is using Hue bridge and the other ZHA, I will let you guess which is ZHA 🤣  both same scene/brightness.

![PXL_20240930_181303860](https://github.com/user-attachments/assets/0cf4b432-5ae1-4f1c-9005-7eed243479d6)
![PXL_20240930_181327344](https://github.com/user-attachments/assets/e5936614-e9fe-47bb-917e-08d1183c4316)

**Comment by dmulcahey** [comment]
Not trying to be a pain but they are different physical devices. Can you do us a favor and add the round one to the hub and see what it looks like at 1%? Thanks!

**Comment by Glennmen** [comment]
@dmulcahey Sure, there are 2 Hue White and Color Ambiance lights in it. I have now reset and moved 1 back to my Hue bridge and applied the same scene.
![PXL_20240930_183423086](https://github.com/user-attachments/assets/0f3edb50-4597-4e32-973e-7e49fcda3512)

Seems to me that this is some weird Hue quirk that they patched in the bridge.

**Comment by Glennmen** [comment]
> see what it looks like at 1%

To avoid confusion in this case I am not talking about `brightness_pct` but `brightness` (value 1-255).

**Comment by puddly** [comment]
Have you tried updating the lamp's firmware to the latest release?

**Comment by Glennmen** [comment]
Now that it was connected to my bridge I updated it to the latest version.
After update I checked if the behaviour was the same on the bridge and confirmed it was.
Then disconnected it from the bridge and reconnect to ZHA (with latest firmware version) and the light it still turned off with the exact same scene/brightness.

**Comment by Glennmen** [comment]
I read a comment somewhere that this also used to be the case with z2m but they patched it.
Went through their code and found this "workaround" 
https://github.com/Koenkk/zigbee-herdsman-converters/blob/13ca5a04abfb6e38a03aee0d3eb4737a18e33071/src/converters/toZigbee.ts#L1160-L1162

**Comment by TheJulianJES** [comment]
Yeah, I think the behavior on the Hue side is intentional. This can't be caused via the HA UI, right? So, any reason you need to be sending `brightness: 1` (instead of 1 % brightness) in a service call?
We could implement a similar workaround to Z2M, but I rarely see this being caused accidentally. I have to see if I can find out *why* this is the case with brightness value 1.

**Comment by Glennmen** [comment]
I was playing around with this project and it has a scene with `brightness: 1` and that is how I went down the brightness rabbit hole 😅  -> https://github.com/Hypfer/hass-scene_presets

Here they mention (old post) it used to also be the case with Hue bridge but Philips/Signify changed it. I cannot find any more information or release notes from Hue about this topic.
https://zigbee2mqtt.discourse.group/t/brightness-issue-in-ha-with-hue/1843/2

**Comment by puddly** [comment]
It's disappointing that Hue is still releasing broken firmware for the device. It should be possible to write a quirk for the device to translate all individual light commands and mutate the brightness but if the bulb is part of a Zigbee group, this isn't behavior that can be corrected in that situation.

Can you upload diagnostic JSON for the device? It'll contain info to uniquely identify the model, including firmware version.

**Comment by Glennmen** [comment]
This is what you requested right? 
[zha-78e336d01f5f64a45cdc42e66a86631f-Signify Netherlands B.V. LCA001-1fc8058a87a652d5b522d6bc753b6cfb.json](https://github.com/user-attachments/files/17210515/zha-78e336d01f5f64a45cdc42e66a86631f-Signify.Netherlands.B.V.LCA001-1fc8058a87a652d5b522d6bc753b6cfb.json)

But I have to add that this is not only the case with this specific model but with every Hue light (that I have ), I see the same behaviour with my non color Hue lights.
I think that z2m also applies the same same patch/quirk to all Philips lights and not only specific models. https://github.com/Koenkk/zigbee-herdsman-converters/blob/13ca5a04abfb6e38a03aee0d3eb4737a18e33071/src/lib/philips.ts#L56
